examples/image_classification/ios/ImageClassifier/Helper/ImageClassifierHelper.swift

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import MediaPipeTasksVision
+
+class ImageClassifierHelper {
+
+  var imageClassifier: ImageClassifier?
+
+  init(model: Model, maxResults: Int, scoreThreshold: Float, runningModel: RunningMode) {
+    guard let modelPath = model.modelPath else { return }
+    let imageClassifierOptions = ImageClassifierOptions()
+    imageClassifierOptions.runningMode = runningModel
+    imageClassifierOptions.maxResults = maxResults
+    imageClassifierOptions.scoreThreshold = scoreThreshold
+    imageClassifierOptions.baseOptions.modelAssetPath = modelPath
+    imageClassifier = try? ImageClassifier(options: imageClassifierOptions)
+  }
+
+  func classify(image: UIImage) -> ImageClassifierHelperResult? {
```

Comment:
Does iOS have a classifyAsync option? I noticed classification is kind of slow from the camera frame, but we don't have that issue on Android because camera frames are classified asynchronously.